Build/Build.cs

```diff
@@ -246,5 +258,22 @@ class Build : NukeBuild
                     (v, path) => v.SetTargetPath(path)));
         });
 
+    Target Spellcheck => _ => _
+        .Executes(() =>
+        {
+            InstallCSpell();
+            RunCSpell();
+        });
+
+    private void InstallCSpell()
```

Comment:
⛏️ I prefer to not have small methods which name literally repeats what the single line is doing.

Reply:
Ok